package.json

```diff
@@ -16,15 +16,16 @@
     "test": "node --test",
     "test:coverage": "node --test --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=lcov.info",
     "test:watch": "node --test --watch",
-    "lint": "eslint src/ test/",
-    "lint:fix": "eslint --fix src/ test/",
+    "lint": "prettier . --check && eslint .",
+    "lint:fix": "prettier --write . && eslint . --fix",
     "precommit": "lint-staged",
```

Comment:
Hmmm ... interestingly, I did push these changes but for some reason this is not showing the update.

Reply:
> This can probably be removed as `lint-staged` is executed using `npx` already, and I don't see this script being used anywhere.

It seems like if I remove the `precommit`, the checks don't run ... 😕 


https://github.com/user-attachments/assets/e4990844-18d1-4773-9111-f817ffb7e32c


By the looks of it, after I ran `npx husky init` things seem to be working w/o the script in there.

Reply:
This is caused by the fact that the `prepare` script that sets up Husky is missing, if you run `npx husky init` it should add this script to the `package.json` or you can [add it manually](https://typicode.github.io/husky/how-to.html#manual-setup).

Reply:
I've fixed that. It should be there.